config/initializers/simple_form.rb

```diff
@@ -1,10 +1,10 @@
 # rubocop:disable Metrics/BlockLength
 SimpleForm.setup do |config|
+  require Rails.root.join('lib/simple_form/error_notification')
```

Comment:
I *think* Rubocop when run on the server might get mad at this and ask for `Rails.root.join('lib', 'simple_form', 'error_notification')` but if it doesn't then NBD

Reply:
> I _think_ Rubocop when run on the server might get mad at this and ask for `Rails.root.join('lib', 'simple_form', 'error_notification')` but if it doesn't then NBD

👍  Updated in 8ba84fa.